### ISSUE TITLE
Resuming playback failure

### ISSUE THREAD

**Issue by Tyzer34** [Tyzer34/plexMusicPlayer#2]
Resuming the song, restarts the song currently being played. Also, due to the memory inconsistency problem, it sometimes returns a NullType object, which is not yet handled.

**Comment by Tyzer34** [comment]
Paused time can be requested using:
```
from flask import request
@ask.on_playback_stopped()
def stopped():
    offset = request.json['context']['AudioPlayer']['offsetInMilliseconds']
    queue.current.set_offset(offset)
    print("Playback stopped at %s milliseconds" % offset)
```

**Comment by Tyzer34** [comment]
Returning the offset in the response, as shown [here](https://developer.amazon.com/blogs/post/Tx1DSINBM8LUNHY/new-alexa-skills-kit-ask-feature-audio-streaming-in-alexa-skills) does not allow the stream to start from that offset. Instead, the stream just does not play anymore.

**Comment by Tyzer34** [comment]
https://forums.developer.amazon.com/questions/52441/audio-streaming-offset-is-not-working.html

**Comment by Tyzer34** [comment]
Fixed in Flask-Ask 0.8.4
